terraform/deploy_new_bod_bastion_ami.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# deploy_new_bod_bastion_ami.sh region workspace_name
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [ $# -eq 2 ]
+then
+    region=$1
+    workspace=$2
+else
+    echo "Usage:  deploy_new_bod_bastion_ami.sh region workspace_name"
+    exit 1
+fi
+
+terraform workspace select "$workspace"
+
+# Strip control characters, then look for the text "id" surrounded by
+# space characters, then extract only the ID from that line.
+#
+# The first sed line has been carefully crafted to work with BSD sed.
+bod_bastion_instance_id=$(terraform state show aws_instance.bod_bastion | \
+                           sed $'s,\x1b\\[[0-9;]*[[:alpha:]],,g' | \
+                           grep "[[:space:]]id[[:space:]]" | \
+                           sed "s/[[:space:]]*id[[:space:]]*= \"\(.*\)\"/\1/")
+
+# Terminate the existing reporter instance
```

Comment:
Verify this comment.

Reply:
Thanks for the sharp 👀 @felddy!  Fixed in https://github.com/cisagov/cyhy_amis/pull/234/commits/16cf850fd7434d9b2ec402a33b23b43ceab859f5.